readme.md

```diff
@@ -152,11 +152,16 @@ This option can also be enabled ad-hoc by running the task like `grunt php:targe
 
 ### open
 
-Type: `boolean`  
+Type: `boolean` or `string`
 Default: `false`
 
 Open the server in the browser when the task is triggered.
 
+This can be one of the following:
+
+- `true`: opens the default server URL ('http://' + hostname + port)
+- a full URL (string): opens that URL in the browser. Useful when testing pages that are not the default one.
```

Comment:
Would be nice to support relative paths too, so you could just do `open: 'index2.php'` instead of having to write the full URL.